databases/week3/make_database.sql

```diff
@@ -0,0 +1,38 @@
+CREATE DATABASE hyf_meal;
+USE hyf_meal;
+CREATE TABLE meal
+(
+	id INT UNSIGNED AUTO_INCREMENT NOT NULL,
+    title VARCHAR(255) NOT NULL,
+    description TEXT,
+    location VARCHAR(255) NOT NULL,
+    event_time DATETIME NOT NULL, -- CHECK( event_time > CURRENT_TIMESTAMP() ) DEFAULT(getdate()),
+    max_reservation INT UNSIGNED NOT NULL DEFAULT('1'),
```

Comment:
This might be a typo, but i think it is more correct to define an integer default like you've done for `number_of_guests` in the reservation table